.github/workflows/test.yml

```diff
@@ -33,5 +30,6 @@ jobs:
         with:
           node-version: '16.x'
       - run: npm install
-      - run: npm run get
       - run: npm run build
+      - run: npm run build:dev
```

Comment:
This adds tests to make sure our example.ts synths and uses terraform to validate